### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

**Comment by SpacePossum** [comment]
:+1:

**Comment by keradus** [comment]
:+1:

Thanks @fabpot !

**Comment by keradus** [comment]
Thank you @fabpot.

File: Symfony/CS/Console/Command/FixCommand.php [hunk 1]
```diff
@@ -269,7 +269,7 @@ protected function configure()
 Create a build file to run ``php-cs-fixer`` on Travis. It's advisable to create a dedicated directory
 for PHP CS Fixer cache files and have Travis cache it between builds.
 
-    <?yml
+    <?yaml
```
Comment (stof) [inline Symfony/CS/Console/Command/FixCommand.php]: why is it ``<?`` ?

Comment (keradus) [inline Symfony/CS/Console/Command/FixCommand.php]: only internal marker for regexp

**Pull request merged**